### PR TITLE
feat: add guides pages

### PR DESCRIPTION
**Pull request by saimonkat**
**Describe what changes this pull request brings**
This PR brings new guide pages

**Screenshots**
<details>
<summary>Index page</summary>

image

</details>
<details>
<summary>Single page</summary>

image

</details>

**References**
[Preview](https://neon-next-git-guide-pages-neondatabase.vercel.app/guides)
[Preview Guide](https://neon-next-git-guide-pages-neondatabase.vercel.app/guides/next-server-actions)

### CODE REVIEW

**Comment by rishi-raj-jain** [comment]
@saimonkat

Glad to see this rise from #1644!

Can you also incorporate the latest guide I've written in place of `content/guides/sample.md` in the commit  https://github.com/neondatabase/website/pull/1644/commits/279138158723ee4d6f1accf9cbf2c74ec4d7b6c8?

**Comment by saimonkat** [comment]
@rishi-raj-jain sure, I'll update tomorrow

**Comment by rishi-raj-jain** [comment]
@saimonkat 

Thanks!

File: content/guides/next-server-actions.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  ```tsx {3,6-16}
+  // File: app/page.tsx
+
+  import { Client } from "pg";
+
+  export default function Page() {
+    async function create(formData: FormData) {
+      "use server";
+      // Create a client instance using `node-postgres`
+      const client = new Client(`${process.env.POSTGRES_URL}`);
```
Comment (rishi-raj-jain) [inline content/guides/next-server-actions.md:85]: the latest commit in #1644 replaces this with DATABASE_URL, please have a look at https://github.com/neondatabase/website/pull/1644/commits/9a091ce2b47ed54e00d899365af2e63f2cbd8a12?

Comment (saimonkat) [reply content/guides/next-server-actions.md:85]: @rishi-raj-jain ok updated!

Comment (rishi-raj-jain) [reply content/guides/next-server-actions.md]: Awesome!

**Comment by saimonkat** [comment]
@rishi-raj-jain I resolved the issue in your latest guide version with https://github.com/neondatabase/website/pull/1671/commits/55dc3435911a45addcb9059330a1fa66506ab01b

File: content/guides/authors/data.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "rishi-raj-jain": {
+    "name": "Rishi Raj Jain",
+    "position": "Developer",
+    "bio": "Previously, Technical Support Engineer @lmsqueezy, Technical CSM @Edgio & Ambassador @storyblok.",
```
Comment (rishi-raj-jain) [inline content/guides/authors/data.json:5]: May we use just "Software Engineer" instead?

Comment (rishi-raj-jain) [reply content/guides/authors/data.json:5]: Or how about "Technical Writer"?

Comment (saimonkat) [reply content/guides/authors/data.json:5]: @rishi-raj-jain sure, this is just mock data, I found about you. Please feel free to send me your real data (name, position, bio, and photo) if you like to change something.

Comment (rishi-raj-jain) [reply content/guides/authors/data.json:5]: All's fine, how about using "Technical Writer" instead of the big description?

Comment (saimonkat) [reply content/guides/authors/data.json]: @rishi-raj-jain ok sure, please check the update

Comment (rishi-raj-jain) [reply content/guides/authors/data.json]: Looks good :D

**Review by americano98 — APPROVED** [review]

**Comment by rishi-raj-jain** [comment]
Currently on https://neon-next-qms06xys4-neondatabase.vercel.app/guides/next-server-actions on scroll the "On this page" section collides with the author section. A particular detail though, not something urgent.

https://github.com/neondatabase/website/assets/46300090/8156b337-7957-47e9-8d43-0b7adb777410

**Pull request merged**